Felienne/hedy
# [FEATURE] Implements quiz syntax highlighting

**Description**
In this PR we implement a (long-awaited, my bad!) feature for syntax highlighting support within the quiz. We also clean-up the template a bit to be (more) correct HTML. For each code block in the question or answer the code is converted into a read-only ace editor with the corresponding language and level syntax highlighting. We did have to perform some not really nice styling as the quiz itself is a very complicated multi-dimensional-sci-fi-complex CSS. Lastly we fix a bug where the results page would crash due to a mathematical calculation on a string.

**Fixes**
This PR fixes #1290

**How to test**
Select any quiz and navigate to a question containing code in either the question or the answers. Verify that the code is correctly shown as a read-only ace editor with the correct syntax highlighting of your language and level.

![Schermafbeelding 2022-03-14 om 14 31 04](https://user-images.githubusercontent.com/38155424/158182779-be96d8ec-8973-42d1-9406-dca34edb4ff8.png)
